reference/1.21/src/main/java/com/example/docs/block/ModBlocks.java

```diff
@@ -18,6 +18,10 @@
 import com.example.docs.block.custom.PrismarineLampBlock;
 import com.example.docs.item.ModItems;
 
+import org.jetbrains.annotations.NotNull;
+
+import java.util.function.Function;
+
```

Comment:
we usually only change stuff in the latest version.. also this seems to be unused.

Reply:
Yes, please dont modify older versions, they exist in an archived state, we should really remove them from settings.gradle in a future PR.